### PR TITLE
Added Turkish translation

### PR DESCRIPTION
**Pull request by badmaan**

### CODE REVIEW

**Comment by Haxk20** [comment]
Hello. Thank you for contributing. Please only translate values that are needed. An example commit is https://github.com/sonyxperiadev/packages_apps_ExtendedSettings/commit/14735e1da75fcbbe291d4591e042af24fcab269d
Only translate the values that are in that commit. The rest should be deleted from the file. Thank you.

**Comment by Haxk20** [comment]
Hello thank you for your contribution. Could you at least provide a brief description for the pull request ?
Other then that it looks good to me.

**Comment by badmaan** [comment]
I'll redo it tomorrow, thanks ❤️.

**Comment by Haxk20** [comment]
> I'll redo it tomorrow, thanks heart.

Do not close the Pull request. You can leave it open just edit it later.

File: res/values-tr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <string name="pref_title_dynres_switch">Dinamik Çözünürlük Ayarı</string>
+    <string name="pref_title_dispcal_switch">Beyaz Nokta Kalibrasyonu</string>
+    <string name="error_connect_to_wifi">Hata: Bir WiFi ağına bağlı olmanız gerekiyor!</string>
+    <string name="pref_title_glove_mode_switch">Eldiven modu</string>
+    <string name="pref_title_glove_mode_switch_summary">Dokunmatik hassasiyetini artırır</string>
+    <string name="dialog_reboot_required">Yeniden başlatma gerekiyor</string>
+    <string name="dialog_reboot_required_summary">Değişikliklerin uygulanması için cihazın yeniden başlatılması gerekiyor!</string>   
+    <string name="dialog_enable_glove_mode">Eldiven modu dokunmaların doğruluğunu azaltabilir. <i>Dokunmayı göster</i> ayarı eldiven modunun açık olup olmadığından haberdar olmanız için etkinleştirilecek. <i>Geliştirici seçenekleri</i> menüsünden devre dışı bırakabilirsiniz.</string>
+    <string name="dialog_disable_glove_mode">Eldiven modunu kapatmak Geliştirici seçenekleri'ndeki <i>Dokunmayı göster</i> ayarını kapatır.</string>
```
Comment (MarijnS95) [inline res/values-tr/strings.xml:17]: Unescaped apostrophe.

File: res/values-tr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <string name="reboot_now">Şimdi yeniden başlat</string>
+    <string name="reboot_later">Daha sonra yeniden başlat</string>
+    <string name="enable">Etkinleştir</string>
+    <string name="cancel">İptal</string>
+    <string name="warning">Uyarı</string>
+
+    <!-- DO NOT TRANSLATE Empty summary for dynamic preferences -->
+    <string name="summary_empty" translatable="false"></string>
```
Comment (MarijnS95) [inline res/values-tr/strings.xml:25]: > `translatable="false"`

Please remove this.

**Comment by MarijnS95** [comment]
@badmaan Again, please DO NOT close pull requests but push the requested changes to the same branch. That way we keep review and discussion in one consolidated place!

**Comment by badmaan** [comment]
I got it, I won't send any PR in future. Just nvm this.

**Comment by Haxk20** [comment]
> I got it, I won't send any PR in future. Just nvm this.

Why ? The pull request is OK. Tho you are closing it while what we want from you to do is to rebase the PR and force push the commit to your master branch.
That way we can keep it clean.
Read a git tutorial about rebasing and force pushing. That should help you get this ready for merging.

Dont take it that we are angry or mad at you. We arent. Again we want to just keep the communication and reviews in 1 place instead of creating new pull request for each single change you do.

**Comment by MarijnS95** [comment]
> I won't send any PR in future

I don't understand what you mean. If my gut feeling is correct and you feel hurt by our review, that's nothing personal. I made [the same mistake before](https://github.com/sonyxperiadev/packages_apps_ExtendedSettings/pull/52) (and many more times locally) and want to prevent your PR from breaking the build for others:

```
[ 98% 557/568] AAPT2 compile out/target/common/obj/APPS/ExtendedSettings_intermediates/flat-res/packages/apps/ExtendedSettings/res/value
FAILED: out/target/common/obj/APPS/ExtendedSettings_intermediates/flat-res/packages/apps/ExtendedSettings/res/values-tr_strings.arsc.flat
/bin/bash -c "out/host/linux-x86/bin/aapt2 compile -o out/target/common/obj/APPS/ExtendedSettings_intermediates/flat-res/packages/apps/ExtendedSettings/res/ --pseudo-localize --legacy packages/apps/ExtendedSettings/res/values-tr/strings.xml"
packages/apps/ExtendedSettings/res/values-tr/strings.xml:17: error: unescaped apostrophe in string
"Eldiven modunu kapatmak Geliştirici seçenekleri'ndeki ".
packages/apps/ExtendedSettings/res/values-tr/strings.xml:17: error: not a valid string.
packages/apps/ExtendedSettings/res/values-tr/strings.xml: error: file failed to compile.
11:39:06 ninja failed with: exit status 1
```

**Comment by badmaan** [comment]
I do understand the problem with this and yes, everyone does mistakes at some point, it's not related to you guys. Fixing this is easy but I don't want to continue to do this. Thank you for helping ❤️.

**Comment by Haxk20** [comment]
@badmaan Is it OK if i fix the commit and push it in seperate PR ? It will be still commit with your name on it. Just that it will be fixed. Is that OK ?

**Comment by badmaan** [comment]
Yeah it's ok.

**Comment by MarijnS95** [comment]
> but I don't want to continue to do this

Any reason for that? You put effort in translating it all, addressing some minor review comments to make it work shouldn't be too much to ask? Feel free to request more guidance, closing a PR doesn't tell us where we can help out with. Luckily that was offered and accepted above.

To give you the full perspective the core contributors on this project have collaboratively decided to do more review of each others work as we run into breaking issues far too often - which are easily prevented it if at least one person has glanced over the changes. This is why we check pull requests - even simple (yet build-breaking) changes like this - rigorously.

**Pull request closed without merging**